python/pip.bzl

```diff
@@ -21,88 +21,11 @@ compile_pip_requirements = _compile_pip_requirements
 package_annotation = _package_annotation
 
 def pip_install(requirements = None, name = "pip", **kwargs):
-    """Accepts a `requirements.txt` file and installs the dependencies listed within.
+    # pip_install is now considered deprecated.
```

Comment:
I think we should throw a deprecation warning message here and leave it for only 1 release. Then we delete pip_install.